### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mccaky**
Updated the README file to include a description of the bump hunter functionality, as requested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,28 @@ hpstr anaVtxTuple_cfg.py -i /nfs/slac/g/hps3/users/bravo/data/physrun2016/7800/h
 ```
 This example will run the standard vertex selection on a data file (to specify that this file is data one has to use the ```-t``` flag and passing 0 will tell hpstr that we are processing MonteCarlo. Plots will be produced according to the selections specified. 
 
+### Bump Hunt Analysis
+
+hpstr also includes the HPS 2019 resoancne search analysis functionality. A brief description of the capabilities of this code, and how to use it, is included here.
```
Comment (pbutti) [inline README.md]: typo -> resonance

File: README.md [hunk 2]
```diff
@@ -71,6 +71,28 @@ hpstr anaVtxTuple_cfg.py -i /nfs/slac/g/hps3/users/bravo/data/physrun2016/7800/h
...
+A resonance search job may be called with the following form:
+
+hpstr bhToys_cfg.py -i ${invariantMassDistro.root} -s ${pathToInvariantMassPlot} -d ${outputDirectory} -m ${mass} -p ${backgroundFitPolynomialOrder} -w ${windowSize} -n 0
+
+The variables correspond to:
+   ${invariantMassDistro.root}: A ROOT file containing the invariant mass histogram that is to be searched.
+   ${pathToInvariantMassPlot}: The path to the invariant mass histogram within the ROOT file.
+   ${outputDirectory}: The directory in which to output the result ROOT ntuple.
+   ${mass}: The mass to probe. This must be in units of MeV.
+   ${backgroundFitPolynomialOrder}: The resonance search uses background fit polynomials of the form 10^(T_n(m)), where T_n(m) is a Chebyshev polynomial of the first kind of order n. This argument specifies n, which may be any value from 0 - 5.
```
Comment (pbutti) [inline README.md:87]: isn't it e^{} ? really 10^ ?

File: README.md [hunk 3]
```diff
@@ -71,6 +71,28 @@ hpstr anaVtxTuple_cfg.py -i /nfs/slac/g/hps3/users/bravo/data/physrun2016/7800/h
...
+   ${windowSize}: The size of the fit window. This is in units of the mass resolution, so a window size of 5 is equivalent to (5 * massResolution(m)).
+
+This will run a test to determine the p-value, signal yield, and other basic values for the resonance search at the specified point. Multiple jobs must be run to analyze multiple points.
+
+In addition to the above mentioned variables, it is also possible to sacle the mass resolution by an arbitrary float by adding the argument "-r ${scalingFactor}".
```
Comment (pbutti) [inline README.md]: typo -> scale

**Review by pbutti — CHANGES_REQUESTED** [review]
Just fix typos. Description looks good. Thanks for this

File: README.md (hunk 2 above)
Comment (cbravo135) [reply README.md:87]: It really is 10^

File: README.md (hunk 1 above)
Comment (mccaky) [reply README.md]: This should be fixed now.

File: README.md (hunk 3 above)
Comment (mccaky) [reply README.md]: This should be fixed now, too.

**Review by cbravo135 — APPROVED** [review]

**Comment by mccaky** [comment]
The two typos you pointed at should be fixed now. Let me know if anything else needs improvement.

**Pull request merged**